roles/openshift_health_checker/test/ovs_version_test.py

```diff
@@ -50,7 +50,7 @@ def test_ovs_package_version(openshift_release, expected_ovs_version):
         openshift_release=openshift_release,
         openshift_image_tag='v' + openshift_release,
     )
-    return_value = object()
+    return_value = {}
```

Comment:
Why this change?

The original is an idiom to say you get the exact same thing when doing comparison.

Reply:
Because now you don't get the exact thing; `execute_module` doctors the result, and it expects a hash returned.

`assert result is return_value` means you get the same hash back... though its contents aren't the same.